fastlane/lib/fastlane/actions/app_store_build_number.rb

```diff
@@ -29,6 +29,7 @@ def self.get_build_number(params)
         app = Spaceship::Tunes::Application.find(params[:app_identifier])
         if params[:live]
           UI.message("Fetching the latest build number for live-version")
+          UI.user_error!("Could not find a live-version of #{params[:app_identifier]} on iTC") unless app.live_version
```

Comment:
Not sure if we do this anywhere else, but a user probably doesn't know what a `live-version` is?